gui/app/main.js

```diff
@@ -318,6 +333,47 @@ const chooseSyncPath = () => {
   sendToMainWindow('registration-done')
 }
 
+const incompatibilitiesErrorMessage = (i) => {
+  const reasons = []
+  const docType = translate(`Helpers ${i.docType}`)
+  if (i.reservedChars) {
+    reasons.push(
+      interpolate(
+        translate('Error {0} names cannot include « {1} » characters'),
```

Comment:
I think that « ... » is used in french, but not in english, where “...” is prefered